magento1-module/app/code/local/Divante/VueStorefrontBridge/controllers/PrzelewyController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+// @codingStandardsIgnoreStart
+require_once 'AbstractController.php';
+require_once __DIR__ . '/../helpers/JWT.php';
+require_once __DIR__ . '/../Model/Payment/Przelewy24.php';
+require_once __DIR__ . '/../../../../community/Dialcom/Przelewy/Model/Payment/Przelewy.php';
+
+/**
+ * Class Divante_VueStorefrontBridge_PrzelewyController
+ *
+ * @category  Divante
+ * @package   VueStoreFrontBridge
+ * @author    Dariusz Oliwa <doliwa@divante.pl>
+ * @copyright 2018-2019 Divante Sp. z o.o.
+ * @license   See LICENSE_DIVANTE.txt for license details.
+ * @link      n/a
+ */
+class Divante_VueStorefrontBridge_PrzelewyController extends Divante_VueStorefrontBridge_AbstractController
+{
+
+    /**
+     * Get active payment methods
+     *
+     * @return string
+     */
+    public function paymentMethodsAction()
+    {
+        try {
+            /**
+             * Payment model
+             *
+             * @var Divante_VueStorefrontBridge_Model_Payment_Przelewy24 $paymentModel
+             */
+            $paymentModel       = new Divante_VueStorefrontBridge_Model_Payment_Przelewy24();
```

Comment:
shouldn't the object be initiated using Mage::getModel fabric instead of `new` keyword?